src/v4.rs

```diff
@@ -1,12 +1,11 @@
 use crate::prelude::*;
-use rand;
 
 impl Uuid {
     /// Creates a random UUID.
     ///
-    /// This uses the [`rand`] crate's default task RNG as the source of random
-    /// numbers. If you'd like to use a custom generator, don't use this
-    /// method: use the `rand::Rand trait`'s `rand()` method instead.
+    /// This uses the [`getrandom`] crate to utilise the operating system's RNG
```

Comment:
@KodrAus should we expose the fact that we are using the `getrandom` crate implementation in the docs?